src/ui-components/index.ts

```diff
@@ -0,0 +1,9 @@
+import a from "./CodapFlowTextInput";
+import b from "./TransformationSubmitButtons";
+import c from "./CodapFlowSelect";
+import d from "./CodapFlowTextArea";
+
+export const CodapFlowTextInput = a;
+export const TransformationSubmitButtons = b;
+export const CodapFlowSelect = c;
+export const CodapFlowTextArea = d;
```

Comment:
Can someone show me a better way to do this? It seems like we should be able to do something like 
```js
import CodapFlowTextInput from "./CodapFlowTextInput";
import TransformationSubmitButtons from "./TransformationSubmitButtons";
import CodapFlowSelect from "./CodapFlowSelect";
import CodapFlowTextArea from "./CodapFlowTextArea";

module.exports = {
  CodapFlowTextInput,
  CodapFlowSelect,
  TransformationSubmitButtons,
  CodapFlowTextArea,
};
```
but that wasn't working.

Reply:
Would something like we did [here](https://github.com/brownplt/duocode/blob/main/src/modules/index.js) work? Not sure if it's better but seems like it worked there.

Reply:
Exactly what I was looking for, thanks!

Reply:
Argh, actually I'm not sure this is quite right since then we can't quite use the `import { name } from "path"` syntax without an extra set of braces.